### PR TITLE
bots: Load a small number of seed tracked known issues

### PR DESCRIPTION
**Pull request by stefwalter**
When doing machine learning, seed the clusters with a small number of known issues. This lets us see whether certain clusters are related to already known issues or not.

 * [x] learn-tests
 * [x] Examine if the tracked items showed up in the cluster attachments or not.

### CODE REVIEW

**Comment by cockpituous** [comment]
learn-tests in progress on cockpit-tasks-1060x.
Log: http://fedorapeople.org/groups/cockpit/logs/learn-tests-9030-20180424-064444/

**Comment by cockpituous** [comment]
learn-tests in progress on cockpit-tasks-0nk9j.
Log: http://fedorapeople.org/groups/cockpit/logs/learn-tests-9030-20180424-064444/

**Comment by cockpituous** [comment]
tests-data command failed: http://fedorapeople.org/groups/cockpit/logs/learn-tests-9030-20180424-064444/

**Comment by cockpituous** [comment]
learn-tests in progress on cockpit-tasks-n5f58.
Log: http://fedorapeople.org/groups/cockpit/logs/learn-tests-9030-20180424-090909/

**Comment by cockpituous** [comment]
learn-tests in progress on cockpit-tasks-kts6h.
Log: http://fedorapeople.org/groups/cockpit/logs/learn-tests-9030-20180424-100202/

**Comment by cockpituous** [comment]
tests-data command failed: http://fedorapeople.org/groups/cockpit/logs/learn-tests-9030-20180424-100202/

**Comment by cockpituous** [comment]
learn-tests in progress on cockpit-tasks-zrtz9.
Log: http://fedorapeople.org/groups/cockpit/logs/learn-tests-9030-20180424-103535/

**Comment by cockpituous** [comment]
tests-data command failed: http://fedorapeople.org/groups/cockpit/logs/learn-tests-9030-20180424-103535/

**Comment by cockpituous** [comment]
learn-tests in progress on cockpit-tasks-0nk9j.
Log: http://fedorapeople.org/groups/cockpit/logs/learn-tests-9030-20180424-164444/

**Comment by cockpituous** [comment]
Task aborted.

**Comment by cockpituous** [comment]
learn-tests in progress on cockpit-tasks-s31sh.
Log: http://fedorapeople.org/groups/cockpit/logs/learn-tests-9030-20180425-044444/

**Comment by cockpituous** [comment]
learn-tests in progress on cockpit-tasks-0z2d6.
Log: http://fedorapeople.org/groups/cockpit/logs/learn-tests-9030-20180425-044444/

File: bots/learn/extractor.py [hunk 1]
```diff
@@ -55,6 +60,21 @@
 def noop(value):
     return value
 
+# Select which items we want to operate on.
+#
+# Because we have so many tracked failures, we need to only bring
+# some of those into our clustering algorithm. We can assume that
+# these are already clusters
+tracked = { }
+def select(item):
```
[the remark this answers is not in the document]
Comment (stefwalter) [reply bots/learn/extractor.py]: It's used in ```bots/learn-tests``` line 10`

[the remark this answers is not in the document]
Comment (larskarlitski) [reply bots/learn/extractor.py]: Oh, of course, I was misreading the import there. Sorry.

**Comment by stefwalter** [comment]
Updated tests-invoke to download the updated model ... since the version number changed.

**Comment by stefwalter** [comment]
I see this error during testing:

```
Traceback (most recent call last):
  File "/build/cockpit/bots/tests-policy", line 382, in <module>
    sys.exit(main())
  File "/build/cockpit/bots/tests-policy", line 98, in main
    output = guessFlake(output, context)
  File "/build/cockpit/bots/tests-policy", line 150, in guessFlake
    model = learn.cluster.load(DATA)
  File "/build/cockpit/bots/learn/cluster.py", line 68, in load
    model = pickle.load(fp, protocol=4)
TypeError: 'protocol' is an invalid keyword argument for this function
```

**Comment by stefwalter** [comment]
Fixed ```pickle.load()``` call.

**Comment by stefwalter** [comment]
Fixed output in tests. Instead of:

```
# Flake likely: 0.4% (clustering). Please file issue if this not a flake
```

put

```
# Flake likely: 0.435 (clustering). Please file issue if this not a flake
```

**Pull request merged**